### DISCUSSION TITLE
Can I use Prisma Rust with mongoldb driver ?

### DISCUSSION THREAD

**Discussion by maxlestage** [Brendonovich/prisma-client-rust#406] [Q&A]
Hello,

I identified your project to develop a student project, at first glance Prisma js integrates nosql, will your driver for rust also allow me to manage a nosql database (Stack: Rust Salvo/Actix)

Good to you and have a good day.

**Comment by Brendonovich** [comment]
PCR does support some MongoDB features like composite types and raw queries, just depends how many mongo-specific features you rely on.
